Blog: Approximate Sparse Retrieval Algorithm (#3977)
opensearch-project/project-website: _community_members/yuye.md

---
name: Yuye Zhu
short_name: yuye
photo: '/assets/media/community/members/yuye.png'
title: 'OpenSearch Community Member: Yuye Zhu'
primary_title: Yuye Zhu
github: yuye-aws
job_title_and_company: 'AWS machine learning engineer with the OpenSearch Project'
personas:
  - author
  - conference_speaker
conference_id:
  - "2025-europe"
session_track:
  - conference_id: "2025-europe"
    name: "Dive Deep on OpenSearch’s Vector Search"
permalink: '/community/members/yuye-zhu.html'
redirect_from: '/authors/yuye/'
---

> **Yuye Zhu** is a Machine Learning Engineer on the OpenSearch Project. His mainly works on neural sparse search, agent framework, text chunking, and retrieval-augmented generation.

Reviewer: ```suggestion
**Yuye Zhu** is a machine learning engineer working on the OpenSearch Project. He primarily works on neural sparse search, agent framework, text chunking, and retrieval-augmented generation.
```
